src/xcode/ENA/ENA/Source/HTTPClient/Service/RestServiceProvider.swift

```diff
@@ -26,9 +31,12 @@ class RestServiceProvider: RestServiceProviding {
 		environment: EnvironmentProviding = Environments(),
 		session: URLSession? = nil
 	) {
+		self.environment = environment
+		self.optionalSession = session
 		self.restService = StandardRestService(environment: environment, session: session)
 		self.cachedService = CachedRestService(environment: environment, session: session, cache: KeyValueCacheFake())
 		self.wifiOnlyService = WifiOnlyRestService(environment: environment, session: session)
+		self.dynamicPinningRestService = DynamicPinningRestService(environment: environment, session: session)
```

Comment:
```suggestion
		self.dynamicPinningService = DynamicPinningService(environment: environment, session: session)
```

Reply:
`DynamicPinningService` might be misleading, it's a new RestService. I think we should keep the name.